content/millennium/r4/scheduling/appointment.md

```diff
@@ -239,11 +244,10 @@ The common [errors] and [OperationOutcomes] may be returned.
 
 In addition, the following errors may be returned:
 
-* Updating an Appointment resource with the incorrect version will result in a `409 Conflict` response.
 * Updating an Appointment resource without sending the `If-Match` header will result in a `412 Precondition Failed` response.
 * Updating an Appointment resource which is currently being modified will result in a `423 Locked` response.
 * If the Appointment resource could not be updated because of an operation that is necessary for the update (eg. encounter association), `424 Failed Dependency` response will be returned.
-* Patching a video visit appointment that has previously been patched for video visit links will result in a `409 Conflict` response.
+* Patching a video visit appointment with add operations that has previously been patched for video visit links will result in a `409 Conflict` response.
 * Patching a video visit appointment with any missing required patch operations will result in a `422 Unprocessable Entity` response.
```

Comment:
There should be a note here about differing patch operations (add + replace) within one PATCH

@MaxPhilips

Reply:
yeah how about

Mixing add and replace patch operations is not supported while patching a Video Visit Appointment and will result in a `123 Asdf` response

Reply:
Added https://github.com/cerner/fhir.cerner.com/pull/385/commits/ab0f399f4c56a6788c04ccd244ad0a9e9453ede9#diff-1d7df8ceb9f0ae8471323516e037c662R251